src/RoadRegistry.BackOffice/Messages/RoadNetworkChangesAccepted.cs

```diff
@@ -11,6 +11,7 @@ public class RoadNetworkChangesAccepted : IMessage, IHaveHash
 {
     public const string EventName = "RoadNetworkChangesAccepted";
     
+    public int Version { get; set; }
```

Comment:
Is this Version used by anything? I would think not because the version is kept in the Changes

Reply:
Nope, not used